lib/measures/BiasedEconomizerSensorOutdoorT/resources/ControllerOutdoorAirFlow_T.rb

```diff
@@ -170,20 +130,20 @@ def econ_t_sensor_bias_ems_main_body(workspace, bias_sensor, controlleroutdoorai
       SET ORI_RETHumRat = RETHumRat, !- <none>
       SET ORI_OAENTH = @HFnTdbW OATmp OAHumRat, !- <none>
       SET ORI_OAHumRat = OAHumRat, !- <none>
-      SET RETENTH = @HFnTdbRhPb RETTmp RETRH PTmp, !- <none>
+	  SET RETENTH = @HFnTdbRhPb RETTmp RETRH PTmp, !- <none>
       SET RETHumRat = @WFnTdbH RETTmp RETENTH, !- Recalculate humidity ratio after the offset
       SET RETRHO = @RhoAirFnPbTdbW PTmp RETTmp RETHumRat, !- Calculate density before offsetting because density is not used by the controller
       SET OAENTH = @HFnTdbW OATmp OAHumRat, !- <none>
     "
   elsif bias_sensor.eql?("OA")
     #########################################################################################################
     main_body = main_body+"
-      SET RETTmp = "+name_cut(econ_choice)+"RETTemp1#{bias_sensor}_#{$faulttype}, !- <none>
-      SET RETHumRat = "+name_cut(econ_choice)+"RETOmega1#{bias_sensor}_#{$faulttype}, !- <none>
-      SET OATmp_ORI = "+name_cut(econ_choice)+"OATTemp1#{bias_sensor}_#{$faulttype}, !- <none>
+	  SET RETTmp = "+econ_short_name+"RETTemp1#{bias_sensor}_#{$faulttype}, !- <none>
+      SET RETHumRat = "+econ_short_name+"RETOmega1#{bias_sensor}_#{$faulttype}, !- <none>
+	  SET OATmp_ORI = "+econ_short_name+"OATTemp1#{bias_sensor}_#{$faulttype}, !- <none>
       SET OATmp = OATmp_ORI"+oa_str_num+"*AF_current_#{$faulttype}_#{oacontrollername}, !- <none>
-      SET OAHumRat = "+name_cut(econ_choice)+"OATOmega1#{bias_sensor}_#{$faulttype}, !- <none>
-      SET PTmp = "+name_cut(econ_choice)+"RETPressure1#{bias_sensor}_#{$faulttype}, !- <none>
+      SET OAHumRat = "+econ_short_name+"OATOmega1#{bias_sensor}_#{$faulttype}, !- <none>
+      SET PTmp = "+econ_short_name+"RETPressure1#{bias_sensor}_#{$faulttype}, !- <none>
```

Comment:
maybe we want to change all of these to use `{econ_short_name}` instead of `"+econ_short_name"` for readability and consistency